backend/app.js

```diff
@@ -1,13 +1,56 @@
-const express = require('express')
-const app = express()
-var cors = require('cors')
-const port = 8000
-app.use(cors())
+// const express = require('express');
+
+// const app = express();
```

Comment:
The next ticket will be changing all the backend code anyways so you can delete this commented out code. But for future reference.. this code should work with the test, you should NOT be making the test pass just by commenting out your real code.